eyeshade/workers/wallet.js

```diff
@@ -189,14 +193,18 @@ exports.workers = {
     async (debug, runtime, payload) => {
       const surveyorId = payload.surveyorId
       const surveyors = runtime.database.get('surveyors', debug)
-      let state
 
       payload.probi = bson.Decimal128.fromString(payload.probi.toString())
-      state = {
+      const $set = underscore.extend({
+        counts: 0,
+        mature: false,
```

Comment:
can we change mature / frozen to use `setOnInsert`. just as a precaution :)

Reply:
that. makes a lot of sense